src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -1361,12 +1455,23 @@ class SceneryWindow final : public Window
         for (size_t tabIndex = 0; tabIndex < _tabEntries.size(); tabIndex++)
         {
             auto widgetIndex = static_cast<WidgetIndex>(WIDX_SCENERY_TAB_1 + tabIndex);
+            auto widgetCoordsXY = ScreenCoordsXY(widgets[widgetIndex].left, widgets[widgetIndex].top);
             auto scgEntry = _tabEntries[tabIndex].GetSceneryGroupEntry();
-            if (scgEntry != nullptr)
+            std::optional<ImageId> imageId = std::nullopt;
+            auto imageOffset = tabIndex == _activeTabIndex ? TabImageOffsetSelected : TabImageOffsetUnselected;
+
+            if (_tabEntries[tabIndex].IsAll())
+            {
+                imageId = ImageId(SPR_G2_ALL_UNSELECTED + imageOffset, colours[1]);
```

Comment:
As above comment would prefer we draw a normal tab background and then our infinity symbol on top.

Reply:
This is done